protected/views/site/help.php

```diff
@@ -1,362 +1,929 @@
 ?>
 
 <div class="content">
-            <div class="container">
-                <section class="page-title-section">
-                    <div class="page-title">
-                        <ol class="breadcrumb pull-right">
-                            <li><a href="/">Home</a></li>
-                            <li><a href="about">About</a></li>
-                            <li class="active">Help</li>
+    <div class="container">
+        <section class="page-title-section">
+            <div class="page-title">
+                <ol class="breadcrumb pull-right">
+                    <li><a href="/">Home</a></li>
+                    <li><a href="about">About</a></li>
+                    <li class="active">Help</li>
+                </ol>
+                <h1 class="page-subtitle">Help</h1>
+            </div>
+        </section>
+        <div class="subsection">
+            <p>The <a href="http://gigadb.org/" target="_blank"><span class="text-italic">GigaDB</span></a> website allows any user to browse, search, view datasets and access data files. If you want to submit a dataset, save searches or be alerted of new content of interest we request that you <a href="/user/create" target="_blank">create an account</a>.</p>
+            <p>A 'Latest news' section will be visible to announce any updates or new features to the database and the RSS feed automatically announces each new dataset release.</p>
+            <p>The <a href="http://gigadb.org/" target="_blank"><span class="text-italic">GigaDB</span></a> homepage allows you to browse datasets by type eg Genomic, Metagenomic, Transcriptomic. Clicking on the DOI (digital object identifier) or image will take you directly to the webpage for the dataset of interest.</p>
+            <p>Alternatively you can use the search functions to find datasets, samples or files of interest.</p>
+        </div>
+        <div class="section">
+            <ul class="nav nav-tabs nav-border-tabs" role="tablist" id="alltabs" aria-label="GigaDB help">
+                <li id="lisearch" role="presentation" class="active"><a href="#search" aria-controls="search" role="tab" data-toggle="tab" aria-selected="true"><span class="text-italic">GigaDB</span> search</a></li>
+                <li id="liguideline" role="presentation"><a href="#guidelines" aria-controls="guidelines" role="tab" data-toggle="tab" aria-selected="false">Submission guidelines</a></li>
+                <li id="livocabulary" role="presentation"><a href="#vocabulary" aria-controls="vocabulary" role="tab" data-toggle="tab" aria-selected="false">Controlled vocabulary</a></li>
+                <li id="liapi" role="presentation"><a href="#interface" aria-controls="interface" role="tab" data-toggle="tab" aria-selected="false">Application programming interface</a></li>
+            </ul>
+            <div class="tab-content">
+                <div role="tabpanel" class="tab-pane active" id="search" aria-labelledby="lisearch">
```

Comment:
these are more or less standard and accessible tabs with logic managed by bootstrap, so nothing was changed in regards to the tabs